constraint/pkg/client/drivers/k8scel/transform/make_vap_objects.go

```diff
@@ -0,0 +1,59 @@
+package transform
+
+import (
+	"fmt"
+
+	templatesv1beta1 "github.com/open-policy-agent/frameworks/constraint/pkg/apis/templates/v1beta1"
+	"github.com/open-policy-agent/frameworks/constraint/pkg/client/drivers/k8scel/schema"
+	"github.com/open-policy-agent/frameworks/constraint/pkg/core/templates"
+	admissionregistrationv1alpha1 "k8s.io/api/admissionregistration/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TemplateToPolicyDefinition(template *templates.ConstraintTemplate) (*admissionregistrationv1alpha1.ValidatingAdmissionPolicy, error) {
+	source, err := schema.GetSourceFromTemplate(template)
+	if err != nil {
+		return nil, err
+	}
+
+	matchConditions, err := source.GetV1Alpha1MatchConditions()
```

Comment:
why do we only convert to the alpha one? backward compat? when should we consider beta and v1?

Reply:
I'd say we'd switch once its possible to generate v1beta1 (or greater) VAP resources.

If we wanted to avoid future breaking changes, I could rename this to `TemplateToV1Alpha1PolicyDefinition()` if you prefer?

Reply:
I'm ok keeping it as is.